### PR TITLE
fix: Read receipts color, change reactions popover colour

### PR DESCRIPTION
**Pull request by przemvs**
## Description

Fix for delivered colour and font size, also changed background colour for reactions popover for light theme. 
Also read receipts is clickable now.

## Checklist

- [ ] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

### Important details for the reviewers

(Delete this section if unnecessary)

- use (x) data
- can be reviewed commit-by-commit
- be sure to look at ...

### CODE REVIEW

File: src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/MessageReactions.styles.ts [hunk 1]
```diff
@@ -46,11 +46,19 @@ export const messageReactionWrapper: CSSObject = {
   maxWidth: '100%',
   marginRight: 'var(--conversation-message-timestamp-width)',
   '.tooltip-content': {
-    backgroundColor: 'var(--gray-95) !important',
+    backgroundColor: 'var(--white) !important',
```
Comment (atomrc) [inline src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/MessageReactions.styles.ts:49]: isn't it supposed to change according to dark mode? 🤔

File: src/style/content/conversation/message-list.less [hunk 2]
```diff
@@ -523,19 +522,36 @@
...
 
 .message-status-read {
   display: flex;
   align-items: center;
+  padding: 0;
+  border: none;
   margin-left: 12px;
-
-  .subline;
+  background: transparent;
```
Comment (atomrc) [inline src/style/content/conversation/message-list.less]: I think you can use the `button-reset-default` class for that

File: src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/MessageReactions.styles.ts (hunk 1 above)
Comment (atomrc) [reply src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/MessageReactions.styles.ts:49]: How was it implemented before? It used to be good on both dark and light mode, right?

Comment (przemvs) [reply src/script/components/MessagesList/Message/ContentMessage/MessageActions/MessageReactions/MessageReactions.styles.ts:49]: Its styled in wire-web-packages, but for reactions colour is different

**Review by atomrc — APPROVED** [review]

**Pull request merged**